cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class PaymentStepPage {
+    private payButton: string;
+    private confirmOrder: string;
+
+    constructor() {
+        this.payButton = "a.bankwire";
+        this.confirmOrder = "#cart_navigation>button>span";
+    }
+
+    public confirmOrdermethod(): void {
```

Comment:
And with that change, can let the name of the method as an action, which can make more sense in terms of readability
```suggestion
    public confirmOrder(): void {
```

Reply:
done @danielgalvis98

Reply:
I think you forgot to do the change here, that's why it failed

Reply:
check the next commit

Reply:
@danielgalvis98  done